distribution/lib/Standard/Base/0.0.0-dev/src/Logging.enso

```diff
@@ -21,6 +23,68 @@ Any.log_message self ~message:Text level:Log_Level=Log_Level.Info =
         Log_Level.Severe -> logger.error message
     self
 
+##
```

Comment:
Why an empty line before the actual comment text starts? This is not the convention we use everywhere.

Normally the convention is: first line provides a short summary (usually 1 sentence) of the method/type, then there is one empty line and any further description. Possibly preceded by tags, but if they are not present we do not keep empty lines at the beginning.